### ISSUE TITLE
[analyzer_plugin] version solving failed after analyzer ^5.0.0

### ISSUE THREAD

**Issue by Carapacik** [dart-lang/sdk#50043]
This tracker is for issues related to:

* Analyzer

Running flutter pub get
```
Because analyzer_plugin 0.11.1 depends on analyzer ^4.1.0 and no versions of analyzer_plugin match >0.11.1 <0.12.0, analyzer_plugin ^0.11.1 requires analyzer ^4.1.0.
So, because analyzer_error_with_freezed depends on both analyzer ^5.0.0 and analyzer_plugin ^0.11.1, version solving failed.
pub get failed (1; So, because analyzer_error_with_freezed depends on both analyzer ^5.0.0 and analyzer_plugin ^0.11.1, version solving failed.)
```
My pubspec.yaml
```yaml
environment:
  sdk: '>=2.17.0 <3.0.0'

dependencies:
  flutter:
    sdk: flutter
  analyzer: ^5.0.0
  analyzer_plugin: ^0.11.1

dev_dependencies:
  flutter_test:
    sdk: flutter
  flutter_lints: ^2.0.1
```
Can you please update the analyzer version inside the package to ^5.0.0?
https://github.com/dart-lang/sdk/blob/58addcf2e6d77813ae175dbdbe1b094f44b84bde/pkg/analyzer_plugin/pubspec.yaml#L10

**Comment by simolus3** [comment]
I've opened https://dart-review.googlesource.com/c/sdk/+/260704 to bump the version in the pubspec.

**Comment by scheglov** [comment]
I published `analyzer_plugin 0.11.2` with https://dart-review.googlesource.com/c/sdk/+/260704 changes.
Strangely I was not able to do this on Friday, some issues with permissions maybe.
